packages/java/tests/spring/react-grid-test/package.json

```diff
@@ -15,11 +15,11 @@
     "@hilla/generator-typescript-plugin-subtypes": "file:../../../../ts/generator-typescript-plugin-subtypes",
     "@hilla/generator-typescript-utils": "file:../../../../ts/generator-typescript-utils",
     "@hilla/react-auth": "file:../../../../ts/react-auth",
-    "@hilla/react-components": "2.2.0-beta2",
+    "@hilla/react-components": "2.2.0-rc1",
     "@hilla/react-crud": "file:../../../../ts/react-crud",
     "@hilla/react-form": "file:../../../../ts/react-form",
     "@polymer/polymer": "3.5.1",
-    "@vaadin/bundles": "24.2.0-beta2",
+    "@vaadin/bundles": "24.2.0-rc1",
```

Comment:
Do you really need the new versions here? Otherwise I'd leave `package.json` out of the PR. Or, if this is needed, also add the lock file.

Reply:
Not really needed, it just updates automatically when starting the IT app. Reverted the change.